packages/client/ui/Dialog/DialogContent.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import * as RadixDialog from '@radix-ui/react-dialog'
+import {twMerge} from 'tailwind-merge'
+import {DialogPortal} from './DialogPortal'
+import {DialogOverlay} from './DialogOverlay'
+import {DialogClose} from './DialogClose'
+
+export const DialogContent = React.forwardRef<
+  HTMLDivElement,
+  React.ComponentPropsWithoutRef<typeof RadixDialog.Content>
+>(({className, children, ...props}, ref) => (
+  <DialogPortal>
+    <DialogOverlay />
+    <RadixDialog.Content
+      ref={ref}
+      className={twMerge(
+        'fixed top-[50%] left-[50%] max-h-[85vh] w-[90vw] max-w-md translate-x-[-50%] translate-y-[-50%] rounded-lg bg-white p-4 shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] focus:outline-none data-[state=open]:animate-contentShow',
```

Comment:
+1 I see that this is copied from radix-ui, which is great for examples, but this is really hard to read. Since we reuse the same box shadows everywhere (https://github.com/ParabolInc/parabol/blob/9ef744ef2d76878f1cec88b6967cb616ff2a1c13/packages/client/styles/elevation.ts) could we create a custom tailwinds shadow so this becomes something like `shadow-dialog` or `shadow-z3`?

Reply:
I see we have modal shadow defined in this file, will copy that in a custom `shadow-dialog`